app/views/provider_interface/sessions/authentication_fallback.html.erb

```diff
@@ -1,30 +1,33 @@
 <% content_for :browser_title, t('page_titles.sign_in') %>
 <% content_for :before_content, govuk_back_link_to(provider_interface_path) %>
 
+<%= govuk_notification_banner(title_text: t('notification_banner.important')) do |notification_banner| %>
+  <% notification_banner.heading(text: 'Temporary login') %>
```

Comment:
I think in the new design there is no "Temporary login" see https://bat-design-history.netlify.app/manage-teacher-training-applications/changes-to-the-sign-in-flow-when-dfe-sign-in-is-unavailable/